install_jupyter_kernel.py

```diff
@@ -0,0 +1,18 @@
+import os
+import sys
+
+legion_dir = os.getcwd() + "/legion"
+sys.path.append(legion_dir)
+from jupyter_notebook.install import driver, parse_args
```

Comment:
This violate the pre-commit format, but I do not know how to fix it.

Reply:
@manopapad Do you have any idea how to fix it?

Reply:
The only issue I see in pre-commit is the unterminated string on line 16, which I already pointed out below.

Reply:
I see another error now:
```
install_jupyter_kernel.py:6:1: E402 module level import not at top of file
```
You would need to move the import statement to the very top of the file. Is there a need to do `sys.path.append` before doing that import?

Reply:
No, it complains about line 6. We can not put `from aaa import bbb` after any normal python code, however, I need to use `sys.path.append(legion_dir) ` to tell python where to find the `jupyter_notebook.install` module.

Reply:
> I see another error now:
> 
> ```
> install_jupyter_kernel.py:6:1: E402 module level import not at top of file
> ```
> 
> You would need to move the import statement to the very top of the file. Is there a need to do `sys.path.append` before doing that import?

Yes, we need it, otherwise Python can not find the `jupyter_notebook.install` module, which is under legion_root/jupyter_notebook

Reply:
Try moving lines 4-6 after line 15, inside the `__name__ == "__main__"` block.

Reply:
fixed